### PR TITLE
A simohsieh/new request uri design

### PR DESCRIPTION
**Pull request by SimoTw**

### CODE REVIEW

**Comment by SimoTw** [comment]
ScreenshotComposeAPIUIUpdate2021-01-27

**Comment by SimoTw** [comment]
ScreenshotAPIRequestNewDesign2021-01-28

File: src/react/components/pages/prebuiltPredict/prebuiltPredictPage.scss [hunk 1]
```diff
@@ -1,11 +1,23 @@
 .formtype-section {
+    width: 50%;
     float: left;
+    padding-right: 5px;
 }
 .locales-section {
+    width: 50%;
     float: right;
+    padding-right: 5px;
+
```
Comment (buddhawang) [inline src/react/components/pages/prebuiltPredict/prebuiltPredictPage.scss:10]: nit: remove blank line.

File: src/react/components/pages/prebuiltPredict/prebuiltPredictPage.tsx [hunk 2]
```diff
@@ -840,12 +841,16 @@ export class PrebuiltPredictPage extends React.Component<IPrebuiltPredictPagePro
...
+
+    private getUpdateRequestURI = () => {
         const { prebuiltSettings } = this.props;
         const { currentPrebuiltType, predictionEndpointUrl } = this.state;
+        let updatedURI = "";
         if (predictionEndpointUrl.includes("?")) {
             const queryString = predictionEndpointUrl.split("?")[1];
             if (!prebuiltSettings || !prebuiltSettings.serviceURI) {
-                this.setState({ predictionEndpointUrl: "" });
             } else {
```
Comment (buddhawang) [inline src/react/components/pages/prebuiltPredict/prebuiltPredictPage.tsx:854]: suggestion: change to `if (prebuiltSettings && prebuiltSettings.serviceURI) {`

**Review by buddhawang — APPROVED** [review]
Approved with minor comments, thanks.

**Pull request merged**